doc/design/design_model_zoo.md

```diff
@@ -0,0 +1,187 @@
+# Model Zoo
+
+SQLFlow model zoo is a place to store model definitions, pre-trained model weights and model documentations. You can directly train, predict, analyze using one of the models using SQLFlow, or you can do model fine-tune, transfer learning to use the model to fit your dataset.
```

Comment:
Our model zoo has some common part with https://www.tensorflow.org/hub, maybe we can take it for reference.

Reply:
Thanks, will take a look~